code-challenges.test.js

```diff
@@ -52,5 +136,35 @@ const hand2 = [5, 5, 3, 3, 4]
 const hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
+// describe("fullHouse", () => {
+//     it("determines whether or not the array is a “full house”.", () => {
+//       expect(fullHouse(hand1)).toBe(true)
+//       expect(fullHouse(hand2)).toBe(false)
+//       expect(fullHouse(hand3)).toBe(false)
+//     });   
+//   });
+// ReferenceError: fullHouse is not defined
+// PASS
+
 
 // b) Create the function that makes the test pass.
+
+// Pseudo Code:
+// Declare a function called fullHouse
+// Set parameter to pass an array of numbers: (numbers)
+// Create a for loop to iterate through the array of numbers
+// Make a conditional statement that determines whether or not the array of numbers is a "full house".
+// Return boolean values
+
+
+// const fullHouse = (numbers) => {
+//     for(let i = 0; i < numbers.length; i++){
+//         if(numbers[0],numbers[1] === numbers[2] && numbers[3] === numbers[4]) {
+//             return true
+//         } else
+//             return false
+//     }
+// }
+// console.log(fullHouse(hand1))
+// console.log(fullHouse(hand2))
+// console.log(fullHouse(hand3))
```

Comment:
Good  work!  One problem, a full house is a one pair and one three of a kind.  Your function only works if the order of the values passed is three of a kind, then pair.  But pair can come first.  I encourage you to rewrite this so it will take either case.